examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,125 @@
+# :code:`sorting_1` so should look like a real unit; the sixth (unit id 5) belongs to :code:`sorting_2`
+# so should look like noise.
+
+sw.plot_unit_templates(analyzer, unit_ids=[0,5])
+
+##############################################################################
+# This is as expected: great! Find out more about plotting using widgets `here <https://spikeinterface.readthedocs.io/en/latest/modules/widgets.html>`_. The labels
+# for our units are then easy to put in a list:
+
+labels = ['good', 'good', 'good', 'good', 'good', 'bad', 'bad', 'bad', 'bad', 'bad']
+
+##############################################################################
+# Step 2: Train our model
+# -----------------------
+#
+# With the labelled data, we can train the model using the :code:`train_model` function.
+# Here, the idea is that the trainer will try several classifiers, imputation strategies and
+# scaling techniques then save the most accurate. To save time, we'll only try one classifier
+# (Random Forest), imputation strategy (median) and scaling technique (standard scaler).
+
+folder = "my_model"
+
+# We will use a list of one analyzer here, we would strongly advise using more than one to
+# improve model performance
+trainer = sc.train_model(
+    mode = "analyzers", # You can supply a labelled csv file instead of an analyzer
+    labels = [labels],
+    analyzers = [analyzer],
+    folder = folder, # Where to save the model and model_info.json file
+    metric_names = None, # Specify which metrics to use for training: by default uses those already calculted
+    imputation_strategies = ["median"], # Defaults to all
+    scaling_techniques = ["standard_scaler"], # Defaults to all
+    classifiers = None, # Default to Random Forest only. Other classifiers you can try [ "AdaBoostClassifier","GradientBoostingClassifier","LogisticRegression","MLPClassifier"]
+    overwrite = True, # Whether or not to overwrite `folder` if it already exists. Default is False.
+)
+
+best_model = trainer.best_pipeline
+
+##############################################################################
+# The above code saves the model in ``my_model/model.skops``, some metadata in
+# ``my_model/model_info.json`` and the model accuracies in ``model_accuracies.csv``
+# in the specified ``folder``.
+#
+# ``skops`` is a file format: you can think of it as a more-secture pkl file. `Read more <https://skops.readthedocs.io/en/stable/index.html>`_.
+#
+# The ``model_accuracies.csv`` file contains the accuracy, precision and recall of the
+# tested models. Let's take a look
+
+accuracies = pd.read_csv(Path(folder) / "model_accuracies.csv", index_col = 0)
+accuracies.head()
+
+##############################################################################
+# Our model is perfect!! This is because the task was *very* easy. We had 10 units; where
+# half were pure noise and half were not.
+#
+# The model also contains some more information, such as which features are importantly.
```

Comment:
Could go into a bit more detail here, what is meant by 'important'? Presumably this is how much each quality metric contributes to the prediction. What do we do with QM that don't contribute much? Shall we drop them?

Reply:
Currently we don't do anything, which is reasonable. There are some issues with dropping features: suppose you have two highly correlated features. Only one is needed for classification and you'll likely end up with one of these features being "important" and one being unimportant. But which one will depend on the random seed of your training. So it requires thought/analysis from the model creator to decide what to do.